zebra-scan/src/scan.rs

```diff
@@ -54,7 +54,7 @@ const INITIAL_WAIT: Duration = Duration::from_secs(15);
 const CHECK_INTERVAL: Duration = Duration::from_secs(30);
 
 /// We log an info log with progress after this many blocks.
-const INFO_LOG_INTERVAL: u32 = 100_000;
+const INFO_LOG_INTERVAL: u32 = 1_000;
```

Comment:
Changed for testing. We can get back to 100_000.

Reply:
```suggestion
const INFO_LOG_INTERVAL: u32 = 10_000;
```